.circleci/config.yml

```diff
@@ -148,11 +148,11 @@ workflows:
             branches:
               only:
                 - /feature\/.*-its/
-      - publish:
-          filters:
-            branches:
-              only:
-                - develop
+#      - publish:
+#          filters:
+#            branches:
+#              only:
+#                - develop
```

Comment:
Publishing fails due to lacking LocationTech credentials. Don't want to be publishing SNAPSHOTS there anyway.